diff --git a/misc/gtkbreeze/main.cpp b/misc/gtkbreeze/main.cpp
index 186d37b..b4dfcaa 100644
--- a/misc/gtkbreeze/main.cpp
+++ b/misc/gtkbreeze/main.cpp
@@ -107,7 +107,7 @@ int setGtk2()
     out << QStringLiteral("widget_class \"*\" style \"user-font\"\n");
     out << QStringLiteral("gtk-font-name=\"Oxygen-Sans Sans-Book 10\"\n"); // matches plasma-workspace:startkde/startkde.cmake
     out << QStringLiteral("gtk-theme-name=\"Orion\"\n");
-    out << QStringLiteral("gtk-icon-theme-name=\"oxygen\"\n"); // breeze icons don't seem to work with gtk
+    out << QStringLiteral("gtk-icon-theme-name=\"breeze\"\n");
     out << QStringLiteral("gtk-fallback-icon-theme=\"gnome\"\n");
     out << QStringLiteral("gtk-toolbar-style=GTK_TOOLBAR_ICONS\n");
     out << QStringLiteral("gtk-menu-images=1\n");
@@ -138,6 +138,8 @@ int setGtk3()
         qCDebug(GTKBREEZE) << "gtkrc3 already exists and is not using oxygen, quitting";
         return 0;
     }
+    QDir dir = QFileInfo(gtkrc3path).dir();
+    dir.mkpath(dir.path());
 
     qCDebug(GTKBREEZE) << "no gtkrc3 file or oxygen being used, setting to new theme";
     QFile gtkrc3writer(gtkrc3path);
@@ -150,7 +152,7 @@ int setGtk3()
     out << QStringLiteral("[Settings]\n");
     out << QStringLiteral("gtk-font-name=Oxygen-Sans 10\n"); // matches plasma-workspace:startkde/startkde.cmake
     out << QStringLiteral("gtk-theme-name=")+gtk3Theme+QStringLiteral("\n");
-    out << QStringLiteral("gtk-icon-theme-name=oxygen\n"); // breeze icons don't seem to work with gtk
+    out << QStringLiteral("gtk-icon-theme-name=breeze\n");
     out << QStringLiteral("gtk-fallback-icon-theme=gnome\n");
     out << QStringLiteral("gtk-toolbar-style=GTK_TOOLBAR_ICONS\n");
     out << QStringLiteral("gtk-menu-images=1\n");
